buildlib/travis-build

```diff
@@ -20,6 +20,16 @@ cd ../build32
 CC=gcc-6 CFLAGS="-Werror -m32" cmake -GNinja .. -DENABLE_RESOLVE_NEIGH=0
 ninja
 
+# aarch64 build to check compilation on ARM 64bit platform
+cd ../build-aarch64
+if ! (which aarch64-linux-gnu-gcc &> /dev/null); then
+   AARCH64_PATH=`find / -type d -name 'gcc-linaro-*-x86_64_aarch64-linux-gnu' -print -quit 2>/dev/null | xargs echo`
+   export PATH=$PATH:${AARCH64_PATH}/bin
```

Comment:
Hmm. cbuild should emulate a bit more of real travis, setup HOME=/home/travis and then have before_script extract the compiler to $HOME/aarch64 and set things so that $HOME/aarch64/bin/ has the compiler binaries. 

Then just refer to $HOME/aarch64 in the build step.

Less risk that unpacking a tarfile from the internet in to the container's / will cause problems!